e2e/scripts/mrt-target-manager.js

```diff
@@ -0,0 +1,322 @@
+const SecureS3Client = require('./aws-s3-client')
+const {Command} = require('commander')
+const fs = require('fs-extra')
+const {MRT_TARGET_DETAILS_FILE} = require('../config')
+
+class MRTTargetManager {
+    constructor(options = {}) {
+        this.bucket = options.bucket
+        this.poolDataFileKey = options.poolDataFileKey
+        this.maxRetries = options.maxRetries || 3
+        this.retryDelay = options.retryDelay || 10000 // 10 seconds
+        this.prNumber = options.prNumber || process.env.GITHUB_PR_NUMBER || null
+        this.branch = options.branch || null
+        this.runId = options.runId || null
+        this.s3Client = new SecureS3Client({
+            region: options.region,
+            readOnly: !process.env.CI,
+            roleArn: process.env.CI ? options.roleArn : options.roleArn, // Don't use role ARN in CI since AWS credentials action handles it
+            roleSessionName: options.roleSessionName || 'LocalDev'
```

Comment:
should this be `localDev`? Does this string mean something in aws pipeline/s3 context at all?

Reply:
Updated the varlues to use more sensible names.